WalletWasabi.Fluent/ViewModels/CoinSelection/Core/CommandViewModel.cs

```diff
@@ -0,0 +1,16 @@
+using System.Reactive;
+using ReactiveUI;
+
+namespace WalletWasabi.Fluent.ViewModels.CoinSelection.Core;
+
+public class CommandViewModel
+{
+	public CommandViewModel(string header, ReactiveCommand<Unit, Unit> command)
+	{
+		Header = header;
+		Command = command;
+	}
+
+	public string Header { get; }
+	public ReactiveCommand<Unit, Unit> Command { get; }
+}
```

Comment:
Suggestion: generalize this class

```csharp
public class CommandViewModel<TIn, TOut>
{
	public CommandViewModel(string header, ReactiveCommand<TIn, TOut> command)
	{
		Header = header;
		Command = command;
	}

	public string Header { get; }
	public ReactiveCommand<TIn, TOut> Command { get; }
}

public class CommandViewModel: CommandViewModel<Unit, Unit>
{
}

```

Also I don't think it should be under `CoinSelection` namespace. Make it something generic.